web/src/assets/svg/caret.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="9" viewBox="0 0 9 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 0L0 9L9 4.5L0 0Z" fill="#586E75"/>
```

Comment:
fwiw, i usually do class="fillStd" on my SVG paths rather than hardcode the color. That makes it easier to change the color, add transitions, etc.

Reply:
good idea, done!